yang/IETF-04/ietf-l3vpn-ntw_sg.yang

```diff
@@ -0,0 +1,1986 @@
+module ietf-l3vpn-ntw_sg {
```

Comment:
Why are we using a new name?

Reply:
No, Just put that name to avoid problems with the stable one. 
I'm just validating that everything is ok. Remove the groupings was a huge task.